README.md

```diff
@@ -20,51 +19,54 @@ And the following sizes:
 - medium
 - large
 
-### Mixins, silent mode and classes
-
-> Mixins and silent mode are only available if you're including o-loading in your project using Bower. If you're using o-loading via the build service, you must use the o-loading classes instead. Both are documented below.
-
-[Full documentation of mixins and variables](http://sassdoc.webservices.ft.com/v1/sassdoc/o-loading)
-
-#### Default loading indicator
-
+## Markup
+In order to display a loading indicator in your product, you will need to supply the theme and the size modifiers in your markup, e.g.:
 ```html
 <div class="o-loading o-loading--light o-loading--small"></div>
 ```
 
+## Sass
+In order to output all of the variations in theme and size of `o-loading`, you'll need to include the following:
 ```scss
-$o-loading-is-silent: false;
 @import 'o-loading/main';
+@include oLoading();
 ```
-
-#### Custom loading indicator
-
+You can also be more selective about which themes or sizes of the loading indicator you wish to output, by using an `$opts` map:
 ```scss
 @import 'o-loading/main';
+@include oLoading($opts: (
+	'colors': ('light'),
+	'sizes': ('medium', 'large')
```

Comment:
I found the available options easy to miss, making them more obvious or just a link to them would be great.